starknet_py/net/schemas/rpc.py

```diff
@@ -111,17 +112,19 @@ class TransactionSchema(Schema):
 
 
 class InvokeTransactionSchema(TransactionSchema):
-    contract_address = Felt(data_key="contract_address", required=True)
-    entry_point_selector = Felt(data_key="entry_point_selector", required=True)
+    contract_address = Felt(data_key="contract_address", load_default=None)
+    sender_address = Felt(data_key="sender_address", load_default=None)
+    entry_point_selector = Felt(data_key="entry_point_selector", load_default=None)
     calldata = fields.List(Felt(), data_key="calldata", required=True)
     nonce = Felt(data_key="nonce", load_default=None)
 
-    @pre_load
-    def preprocess(self, data, **kwargs):
-        return data
-
     @post_load
     def make_transaction(self, data, **kwargs) -> InvokeTransaction:
+        data["contract_address"] = data.get("contract_address") or data.get(
+            "sender_address"
+        )
```

Comment:
Perhaps we should also make sure that `data["contract_address"]` is not None? (not that it's allowed in spec)